chipy_org/settings.py

```diff
@@ -156,7 +156,6 @@ def env_list(key, defaults=None, delimiter=","):
     "django.middleware.common.CommonMiddleware",
     "django.middleware.csrf.CsrfViewMiddleware",
     "django.contrib.auth.middleware.AuthenticationMiddleware",
-    "django.contrib.auth.middleware.SessionAuthenticationMiddleware",
```

Comment:
In case you, like me, were wondering: https://docs.djangoproject.com/en/3.0/releases/2.0/#miscellaneous

Reply:
Yeah, you can imagine my surprise when that threw an error.  😉

-